app/assets/javascripts/discourse/tests/acceptance/topic-test.js

```diff
@@ -240,6 +240,12 @@ acceptance("Topic", function (needs) {
     assert.ok(exists(".category-moderator"), "it has a class applied");
     assert.ok(exists(".d-icon-shield-alt"), "it shows an icon");
   });
+
+  test("Suspended user posts", async function (assert) {
+    await visit("/t/topic-from-suspended-user/54077");
+
+    assert.ok(exists(".suspended"), "it has a class applied");
```

Comment:
I feel like the query is too wide here. I think we should scope it down to the post which we expect to be suspended.